include/mwatcher.hpp

```diff
@@ -18,31 +18,29 @@ class drive_info;
 
 #include <QThread>
 
-class MediaWatcher: public QThread
-{
-	Q_OBJECT
+class MediaWatcher : public QThread {
+  Q_OBJECT
 
 public:
-	MediaWatcher(device *qdev);
-	~MediaWatcher();
+  MediaWatcher(device *qdev);
```

Comment:
Codacy found an issue: [Class 'MediaWatcher' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)